nextcloudappstore/core/templates/nav.html

```diff
@@ -41,6 +41,11 @@
                 <li class="nav-link">
                     <a href="{% url 'home' %}">{% trans 'Apps' %}</a>
                 </li>
+                {% if request.user.is_authenticated %}
```

Comment:
is is_authenticated already a property? if so we should also convert the others to obtain django 1.10 compatability